### PR TITLE
[test] Add API calls in Intel Advisor Roofline check

### PR DESCRIPTION
**Pull request by jgphpc**

### CODE REVIEW

**Comment by jgphpc** [comment]
@jenkins-cscs retry all

File: cscs-checks/tools/profiling_and_debugging/intel_advisor_roofline.py [hunk 1]
```diff
@@ -26,52 +26,68 @@ class IntelRooflineTest(rfm.RegressionTest):
...
+        G3_AOS_SCALAR: gflops,  2.79 arithmetic_intensity', 0.166 380ms <- slow
+        G3_AOS_VECTOR: gflops,  3.79 arithmetic_intensity', 0.125 143ms
+        G3_SOA_SCALAR: gflops,  2.79 arithmetic_intensity', 0.166 351ms
+        G3_SOA_VECTOR: gflops, 10.62 arithmetic_intensity', 0.166  57ms <- fast
     '''
     def __init__(self, repeat, toolsversion, datalayout):
         super().__init__()
         self.descr = 'Roofline Analysis test with Intel Advisor'
+        # advisor/2019 is failing on dom ("Exceeded job memory limit")
+        # https://webrt.cscs.ch/Ticket/Display.html?id=36087
```
Comment (victorusu) [inline cscs-checks/tools/profiling_and_debugging/intel_advisor_roofline.py]: @jgphpc, sorry my lack of knowledge about the issue, but didn't you solve the memory limit on #814?

**Comment by vkarak** [comment]
@jgphpc Please fix the conflicts and then we can merge it.

**Comment by vkarak** [comment]
@jgphpc The test still fails on Dom:

```
  * Reason: sanity error: pattern `loop complete.' not found in `rfm_IntelRooflineAdvisorTest_100000_597843_G3_AOS_SCALAR_job.out'
```

**Comment by vkarak** [comment]
@jenkins-cscs retry dom

**Pull request merged**